CHANGELOG.md

```diff
@@ -39,6 +39,9 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 
 - Fixed importing from a group different than the authenticated user. (#13)
 
+- Changed to use ginutil.WriteProblemError instead of directly using
+  gin.Context methods. (#16)
```

Comment:
nit: "But what does this mean for the end user?"

Your changes here actually changes the response structures when there's an error. Suggest specifying that, such as:

```suggestion
- Changed to return IETF RFC-7807 compatible problem responses on failures
  instead of solely JSON-formatted strings. (#16)
```